internal/sessiontoken/sessiontoken.go

```diff
@@ -415,6 +416,8 @@ func (s *SessionToken) promptAuthentication(da *deviceAuthorization) {
 `
 
 	fmt.Fprintf(os.Stderr, prompt, qrCode, da.VerificationURIComplete)
+
+	brwsr.OpenURL(da.VerificationURIComplete)
```

Comment:
Confirmed, we'll need this gated around a CLI flag. I'd follow the pattern for setting up the QR code CLI flag in https://github.com/okta/okta-aws-cli/blob/master/cmd/root/root.go#L111-L117 , something like:
```go
		{
			name:   "open-browser",
			short:  "b",
			value:  false,
			usage:  "Automatically open the activation URL with the system web browser",
			envVar: "OPEN_BROWSER",
		},
```

Then in `Config` struct https://github.com/okta/okta-aws-cli/blob/master/internal/config/config.go#L32-L44 add:
```
	OpenBrowser bool
```

Also, follow the rest of the pattern we did for qr code in `NewConfig`:
https://github.com/okta/okta-aws-cli/blob/master/internal/config/config.go#L50-L104

Finally, in sessiontoken.go

```go
	if s.config.OpenBrowser {
		brwsr.OpenURL(da.VerificationURIComplete)
	}
```

The feature will also need to be documented in the README, again, just follow the pattern seen with the QR code CLI flag:
https://github.com/okta/okta-aws-cli/blob/master/README.md